app/serializers/api/participant_serializer.rb

```diff
@@ -0,0 +1,103 @@
+module API
+  class ParticipantSerializer < Blueprinter::Base
+    identifier :ecf_id, name: :id
+    field(:type) { "npq-participant" }
+
+    class AttributesSerializer < Blueprinter::Base
+      exclude :id
+
+      field(:email)
+      field(:full_name)
+      field(:trn, name: :teacher_reference_number)
+      field(:updated_at)
+
+      view :v1 do
+        field(:npq_courses) do |object, options|
+          applications(object, options).map { |application| application.course.identifier }
+        end
+
+        field(:funded_places) do |object, options|
+          applications(object, options).map do |application|
+            {
+              npq_course: application.course.identifier,
+              funded_place: application.funded_place,
+              npq_application_id: application.ecf_id,
+            }
+          end
+        end
+      end
+
+      view :v2 do
+        field(:npq_enrolments) do |object, options|
+          applications(object, options).map do |application|
+            {
+              course_identifier: application.course.identifier,
+              schedule_identifier: application&.schedule&.identifier,
+              cohort: application.cohort&.start_year&.to_s,
+              npq_application_id: application.ecf_id,
+              eligible_for_funding: application.eligible_for_funding,
+              training_status: application.training_status,
+              school_urn: application.school&.urn,
+              targeted_delivery_funding_eligibility: application.targeted_delivery_funding_eligibility,
+              funded_place: application.funded_place,
+            }
+          end
+        end
+      end
+
+      view :v3 do
```

Comment:
in v3 email is only in the enrolments not in the higher level attributes, if you can please amend that, thanks